pom.xml

```diff
@@ -248,13 +274,6 @@
         <groupId>org.jenkins-ci.tools</groupId>
         <artifactId>maven-hpi-plugin</artifactId>
         <configuration>
-          <!-- with the default setting seems that an older version of jackson from core is used
-            java.lang.NoSuchMethodError: com.fasterxml.jackson.core.JsonFactory.requiresPropertyOrdering()Z
-            at com.fasterxml.jackson.databind.ObjectMapper.<init>(ObjectMapper.java:549)
-            at com.fasterxml.jackson.databind.ObjectMapper.<init>(ObjectMapper.java:460)
-            at io.fabric8.kubernetes.client.utils.Serialization.<clinit>(Serialization.java:37)
-          -->
-          <pluginFirstClassLoader>true</pluginFirstClassLoader>
```

Comment:
Be sure to test this live, not just `mvn hpi:run`.

Reply:
Didn't see any problem. Looking at the git blame for this, I see this was added back in the day where jackson was packaged directly in the plugin.